### PR TITLE
1.2 Russian translation alternative

### PR DESCRIPTION
**Pull request by emartynov**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #322.

I know about existing and approved #335 (which I'm going to review after). I started work a quite long time ago so a logical step for me was just to finish remaining parts. 

My PR is focused ONLY on updating [the changes in the English 1.2 version](https://github.com/OWASP/owasp-masvs/compare/1.1.4...master) including images and CHANGELOG. 

Please suggest what to do with PR next.

### CODE REVIEW

File: Document-ru/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 1]
```diff
@@ -25,6 +25,8 @@
 | **1.8** | MSTG‑ARCH‑8 | Существует явная политика управления криптографическими ключами (если они есть) и их жизненным циклом. В идеале политика соответствует стандарту управления ключами, например, NIST SP 800-57. |   | ✓ |
 | **1.9** | MSTG‑ARCH‑9 | Существует механизм принудительных обновлений мобильного приложения. |   | ✓ |
 | **1.10** | MSTG‑ARCH‑10 | Безопасность заложена во все этапы жизненного цикла разработки программного обеспечения. |   | ✓ |
+| **1.11** | MSTG‑ARCH‑11 | Существует и эффективно применяется ответственная политика раскрытия информации. |  | ✓ |
+| **1.12** | MSTG‑ARCH‑12 | Приложение должно соответствовать законам конфиденциальности и регулированию. | ✓ | ✓ |
```
Comment (RedHotHub) [inline Document-ru/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]: 1.12 Приложение должно соответствовать законам о защите персональных данных.
Maybe this is the Russian analog of "The app should comply with privacy laws and regulations."?

File: Document-ru/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 2]
```diff
@@ -23,19 +23,21 @@
...
+| **2.10** | MSTG‑STORAGE‑10 | Приложение не хранит чувствительные данные в памяти дольше, чем необходимо, и полностью удаляет их из памяти после работы с ними. |  | ✓ |
+| **2.11** | MSTG‑STORAGE‑11 | Приложение требует от пользователя минимальную настройку доступа к устройству, такую, как установку пин-кода на устройство. |  | ✓ |
+| **2.12** | MSTG‑STORAGE‑12 | Приложение информирует пользователя о персональных данных, которые оно обрабатывает, а также о лучших практиках безопасности, которым должен следовать пользователь при использовании приложения. |  | ✓ |
+| **2.13** | MSTG‑STORAGE‑13 | Конфиденциальные данные локально не должны храниться на мобильном устройстве. Вместо этого необходимые данные должны получаться с сервера и храниться только в памяти. |  | ✓ |
+| **2.14** | MSTG‑STORAGE‑14 | Если конфиденциальные данные все же требуется хранить локально, они должны быть зашифрованы с помощью ключа, полученного из аппаратного хранилища, которое требует проверки подлинности. |  | ✓ |
+| **2.15** | MSTG‑STORAGE‑15 | Локальное хранилище приложения должно быть стерто после излишнего количества неудачных попыток аутентификации. |  | ✓ |
```
Comment (RedHotHub) [inline Document-ru/0x07-V2-Data_Storage_and_Privacy_requirements.md]: Correct. But "...после излишнего количества неудачных попыток..." sounds a bit hard.
Maybe something like this: "...после превышения допустимого количества неудачных попыток..."?

File: Document-ru/0x06-V1-Architecture_design_and_threat_modelling_requireme.md (hunk 1 above)
Comment (emartynov) [reply Document-ru/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]: Why wait - I will apply it.

File: Document-ru/0x07-V2-Data_Storage_and_Privacy_requirements.md (hunk 2 above)
Comment (emartynov) [reply Document-ru/0x07-V2-Data_Storage_and_Privacy_requirements.md]: Applied.

**Comment by kysokzla** [comment]
U translated everything i have already translated? Why u did not commit it earlier? I made the double work? Or you?

**Comment by kysokzla** [comment]
No alternative should exist PR's should be merged in order. First we merge mine then u pull it make marge and after that we merge ur edited PR.

**Comment by emartynov** [comment]
@kysokzla you can see that my first commits landed before you started translation in November. It is some double work indeed.

I don't mind completely vanish my PR if another one satisfies me.

Next time I suggest you to ping person who is assigned for the task to check if some work already started/in progress.

After a night, I feel that my PR should go first and your parts should be consolidated in the PR for improvements the old parts. It is mostly because my PR is only focused on the 1.2 changes.

**Comment by sushi2k** [comment]
Hi @kysokzla, @emartynov, @RedHotHub. Apologies for the delay to come back to you and thanks for your great work on the Russian translation!
I will merge this commit, as it covers more changes than the one from Maxim and as the issue was also assigned to Eugen in the first place. If there is anything missing here that was only in your PR @kysokzla, please feel free to add with another PR.
Thanks again everybody!

**Pull request merged**